Src/Base/AMReX_FPC.cpp

```diff
@@ -100,6 +102,7 @@ FPC::Native32RealDescriptor ()
     defined(__ppc__) || \
     defined(__ppc64__) || \
     defined(_SX)   || \
+    defined(__LITTLE_ENDIAN__) || \
```

Comment:
yes, you are right.  I did it correctly in the first place, but not in the other two...

Will fix now.